readme.md

```diff
@@ -248,12 +248,14 @@ test.only('will be run', t => {
 
 ### Skip-tests
 
-Skip-tests are shown in the output as skipped but never run.
+Skip-tests are shown in the output as skipped but never run. Skip-tests can be written without specifying the test function. This can be useful for planning future tests.
```

Comment:
If I'm understanding things correctly this needs to be changed to reflect `test.todo` right?